README.md

```diff
@@ -1 +1,37 @@
-# code-runner
\ No newline at end of file
+Code Runner: An Environment to run your code
+===================================================
+Code Runner is a web application to run code snippet or code files on a docker environment with code specific configurations.
+
+This application is mainly designed for running students' assignments for different courses.
+
+## Features
+- It has two ends for users - 
```

Comment:
what are "ends"?

Reply:
I meant two user-ends. But I changed it to roles as suggested by you